packages/frontend/app/pods/components/pod/chat/chat-entry/component.ts

```diff
@@ -77,9 +78,14 @@ export default class ChatEntry extends Component<IArgs> {
   }
 
   @action onInput(event: KeyboardEvent) {
+    const EMOJI_REGEX = /:[^:]+:/g;
```

Comment:
Should this regex be pulled out of the class, since it seems to be static?

Reply:
thoughts on this?

Reply:
Seems reasonable.

- [ ] Pull out emoji regex

Reply:
Seems reasonable